spaces/dfi/skin.scss

```diff
@@ -0,0 +1,9 @@
+.dfi {
```

Comment:
This look like the default skin, if you don't have a custom skin you can remove this file.